lib/karma-webpack.js

```diff
@@ -65,7 +83,7 @@ function configToWebpackEntries(config) {
 
   const webpackEntries = {};
   filteredFiles.forEach((filePath) => {
-    webpackEntries[path.parse(filePath).name] = filePath;
+    webpackEntries[getPathKey(filePath)] = filePath;
```

Comment:
Why do we need this?
Can't we just use `filePath` as the attribute key?

Reply:
this is the key that webpack uses internally... it does not like '/' in it...
also it is the url that get's displayed when you build... so if that is too long it's quite annoying :see_no_evil: